ms-authentication/Makefile

```diff
@@ -18,7 +18,7 @@ docker:
 		-t $(MICROSERVICE):dev \
 		.
 
-gobuild: tidy
+gobuild-authentication: tidy
```

Comment:
These are in the specific service make files, why do they need to have the specific service name?